### PR TITLE
KEYCLOAK-16924 Revising Release Notes to meet Red Hat standards

### PR DESCRIPTION
**Pull request by andymunro**
@pdrozd One more document for review.  The changes are very minor.

### CODE REVIEW

File: release_notes/topics/product/7_4_final.adoc [hunk 1]
```diff
@@ -47,7 +50,9 @@ Administrators can see all credentials of a particular user and some public meta
 
 Users can see all their credentials in the Account Console and add or delete credentials. See the section on the New Account Console, which is a Technology Preview feature. The currently supported account console, the User Account Service, does not support this feature. It supports OTP in a similar way to the previous RH-SSO version.
 
-See link:{adminguide_authentication_flow_link}[{adminguide_authentication_flow_name}] in the link:{adminguide_link}[{adminguide_name}] for more details.
+[role="_additional-resources"]
+.Additional resources
+* link:{adminguide_authentication_flow_link}[{adminguide_authentication_flow_name}].
```
Comment (nehachopra27) [inline release_notes/topics/product/7_4_final.adoc]: @andymunro: As per prior version, referring the link:{adminguide_link}[{adminguide_name}] here would be useful.

**Review by nehachopra27 — CHANGES_REQUESTED** [review]

**Comment by andymunro** [comment]
Thanks, @nehachopra27.  I have made that change.

**Comment by andymunro** [comment]
@nehachopra27 Please let me know when you approve this or if you request more changes.

File: release_notes/topics/product/7_4_final.adoc [hunk 2]
```diff
@@ -100,9 +108,16 @@ standalone -Dkeycloak.profile.feature.account_api=enabled -Dkeycloak.profile.fea
...
 When you go to Manage Account, you will see the new Account Console.  Here is a sample screen:
 
-image:{project_images}/device-activity.png[]
+.Device activity
+image:{project_images}/device-activity.png[Device activity]
+
+This new console is based on React and PatternFly 4.  It allows you to use PatternFly CSS variables for easy styling.  It also allows you to remove pages and add your own pages.  Full documentation will be provided at a future release.
 
-This new console is based on link:https://reactjs.org/[React] and link:https://www.patternfly.org/v4/[PatternFly 4].  It allows you to use link:https://www.patternfly.org/v4/developer-resources/global-css-variables/#global-variables[PatternFly CSS variables] for easy styling.  It also allows you to remove pages and add your own pages.  Full documentation will be provided at a future release.
+.Additional Resources
```
Comment (nehachopra27) [inline release_notes/topics/product/7_4_final.adoc]: The casing of the title "Additional Resources" is different to others "Additional resources". I am not sure about which one is correct as per the style guide.

File: release_notes/topics/product/7_4_final.adoc [hunk 3]
```diff
@@ -47,7 +50,9 @@ Administrators can see all credentials of a particular user and some public meta
 
 Users can see all their credentials in the Account Console and add or delete credentials. See the section on the New Account Console, which is a Technology Preview feature. The currently supported account console, the User Account Service, does not support this feature. It supports OTP in a similar way to the previous RH-SSO version.
 
-See link:{adminguide_authentication_flow_link}[{adminguide_authentication_flow_name}] in the link:{adminguide_link}[{adminguide_name}] for more details.
+[role="_additional-resources"]
+.Additional resources
+* link:{adminguide_authentication_flow_link}[Server Administration Guide]
```
Comment (nehachopra27) [inline release_notes/topics/product/7_4_final.adoc:55]: The link it is redirecting to is https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.4/html-single/server_administration_guide/index#_authentication-flows which is not redirecting to the corresponding topic. Please change it to /index#authentication-flows at the end

File: release_notes/topics/product/7_4_final.adoc (hunk 2 above)
Comment (andymunro) [reply release_notes/topics/product/7_4_final.adoc]: Thanks, @nehachopra27 It should be lower case always

File: release_notes/topics/product/7_4_final.adoc (hunk 3 above)
Comment (andymunro) [reply release_notes/topics/product/7_4_final.adoc:55]: Okay, I redefined adminguide_authentication_flow_link in document-attributes-product.adoc

**Review by nehachopra27 — COMMENTED** [review]
Thanks @andymunro , changes look good to me

**Review by nehachopra27 — APPROVED** [review]

**Review by pdrozd — APPROVED** [review]
Approved based on Neha's review

**Pull request merged**